config/rbac/cluster_monitoring_view_prometheus_k8s_role_binding.yaml

```diff
@@ -0,0 +1,12 @@
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: prometheus-k8s-cluster-monitoring-view
+subjects:
+- kind: ServiceAccount
+  name: prometheus-k8s
+  namespace: openshift-storage
```

Comment:
```suggestion
```
It can be dropped as we expect OLM to fill it based on namespace where operator gets deployed.

Reply:
Dropped